### DISCUSSION TITLE
[Unstable] Cracked ruin walls are almost invisible

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14659] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Cracked ruin walls are almost invisible:
* "Draw behind walls" lights don't illuminate them
* Shadow casting lights (like diving suit light) only shows their edge

Examples:

![image](https://github.com/user-attachments/assets/a26cb5dd-0946-43f8-85c7-e73f37609827)
![image](https://github.com/user-attachments/assets/ae406903-df1e-4fdf-b35d-7812f02ef8e2)
![image](https://github.com/user-attachments/assets/887c2a14-a2ce-4e09-ba5d-00d5e905372c)

**The width/height of the collider must be thinner than the sprite itself**

This wall sprite is larger than its collision box, so you can see deep into the sprite
![image](https://github.com/user-attachments/assets/51ae8ab5-653e-4bcc-a30a-580c9b32dcf0)

So for example:
Height property should be half the Y of the sprite

```xml
  <RuinWallHorizontalWeakened name="" identifier="RuinWallHorizontalWeakened" descriptionidentifier="resizeablewallwithcollisiondetection" category="Alien" castshadow="true" width="512" height="80" resizehorizontal="true" body="true" scale="0.5" health="600" noaitarget="true" damagesound="StructureAlien">
    <sprite texture="Content/Map/AlienWallSet1.png" sourcerect="511,1274,512,159" depth="0.06" spritecolor="255,0,0,255" />
  </RuinWallHorizontalWeakened>
```

### Reproduction steps

Try finding a cracked wall I dare you

### Version

v1.6.4.0 (unstable)

**Comment by Regalis11** [comment]
Addressed in our private repo, coming up in the next unstable build
